pages/api/optimus-metrics.js

```diff
@@ -86,11 +86,6 @@ export default async function handler(req, res) {
 
     if (isCacheExpired) {
       fetchLatestMetric().catch(console.error);
-
-      // Returns cached data, even expired
-      if (cachedData) {
-        return res.status(200).json(cachedData.data);
-      }
```

Comment:
no need to remove it - we could still show the cached data, it'll be refetched anyway if stale, see this: https://valory-workspace.slack.com/archives/C046R84E056/p1749659521162749?thread_ts=1749482255.353949&cid=C046R84E056

so with this PR could just set 0.00 to the data/optimus-metrics-cache.json
sorry for confusion